src/Sylius/Bundle/CurrencyBundle/Templating/Helper/CurrencyHelper.php

```diff
@@ -18,9 +18,9 @@
 
 class CurrencyHelper extends Helper implements CurrencyHelperInterface
 {
-    public function convertCurrencyCodeToSymbol(string $code): string
+    public function convertCurrencyCodeToSymbol(string $code, ?string $displayLocale = null): string
```

Comment:
it's used in sylius_currency_symbol Twig extension.
On the whole project, this twig extension is only used in form_theme, but of course could be used in Plugins and userland projects.

Note: I'm wondering if this is the same case with SF6 and maybe this is just my intl locale in PHP.ini